### PR TITLE
[WHAT IS MODULAR] Bone weaponry buff is back in its bad form

### PR DESCRIPTION
**Pull request by Stalkeros2**
## About The Pull Request
Fairly self-descriptive: bone axe, bone knife and bone sword were buffed. Here's the list:
Bone axe's wielded damage bonus against lavaland mobs has been added, increasing damage against those to 32;
Bone spear's armor_penetration and damages are left as is, with a bonus damage against fauna, increasing its damage to 20;
Bone sword's attack speed increased by 20%, effectively increasing its DPS to 24 or so;

## How This Contributes To The Skyrat Roleplay Experience
tl;dr
Generally adding some more niches for bone weaponry so the spear isn't the only one worth using.

## Changelog

:cl: Stalkeros
qol: Bone weaponry (axe and spear) got anti-mob damage buffs; bone sword became 20% faster.
/:cl:

### CODE REVIEW

**Comment by Stalkeros2** [comment]
Meanwhile, I'm finishing my goddamn event and trying to finalise this mess into a component.

**Comment by Stalkeros2** [comment]
No.

**Comment by Gandalf2k15** [comment]
why?

**Comment by dawsonkeyes** [comment]
> why?

good question!

**Comment by jjpark-kb** [comment]
Ill try to add a factional damage component...
